src/main/java/com/softserve/academy/spaced/repetition/controller/dto/impl/DeckPrivateDTO.java

```diff
@@ -11,7 +12,7 @@
 public class DeckPrivateDTO extends DTO<Deck> {
     public DeckPrivateDTO(Deck deck, Link link) {
         super(deck, link);
-        add(linkTo(methodOn(DeckController.class).getCardsByDeck(getEntity().getId())).withRel("cards"));
+        add(linkTo(methodOn(CardController.class).getCardsByDeck(getEntity().getId())).withRel("cards"));
```

Comment:
Not sure, but maybe better to build links inside of controller methods

Reply:
it`s link for another resources

Reply:
I understand, i mean, that this link should be build inside of controller method.